### ISSUE TITLE
Discussion on contact save and close feature

### ISSUE THREAD

**Issue by cagulas** [OpenSRP/opensrp-client-anc#170]
The contact save and exit feature (https://github.com/OpenSRP/opensrp-client-anc/issues/133) was implemented a bit differently than intended, but the way it was implemented probably works better and is a safer option, but we may need to update the labels on the buttons now to better match the actual functions. 

Originally, the intention was for the discard button to discard all data in the contact (so that the user starts again from a blank contact the next time). It was actually implemented whereby only the most recent changes would be discarded. E.g. if you start a contact and save and exit then go back in again and make more changes, the discard button only discards your most recent changes and not any previous data.

This functionality is fine, but we probably want to update the button labels to make this more clear.

Right now, the button labels are:

![screenshot_20190114-110546](https://user-images.githubusercontent.com/3488986/51101692-b0f3d000-17ec-11e9-92d6-88c3a0546918.png)

I propose we update them to:
Save
All changes will be saved.

Discard
Exit without saving changes.

@rowo Can you weigh in on this please before I assign to an engineer?

**Comment by rowo** [comment]
@cagulas Cool, so it's more of a draft until you submit and you're only allowed one draft at a time. I'm not sure how you discard a draft and start a new contact, but I guess that would be an edge case and the workaround is just changing previous inputs.

ODK Collect's popup says the equivalent of: 

**Exit Contact** (the dialog header)
- Save changes
- Ignore changes
- Cancel (shows up in lower right not as buttons)

I think these ODK labels work well since they are short and provide the needed info while also supporting both exiting a new contact and an existing draft contact.

**Comment by cagulas** [comment]
@rowo Great, thanks. I'm ok with your proposal. I'll close this issue and create a new one for implementation.

Just to clarify on the Cancel button, are you ok with keeping the Cancel as is, as in the screenshot above? Or do you want to change it?

**Comment by rowo** [comment]
@cagulas Yes, Cancel as is is good, though I believe Google Material design guidelines says it should be medium weight and it kind of looks like bold weight.
